### DISCUSSION TITLE
情人节更新（2023）

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#1635] [通知]
## 更改记录

**特别感谢 @ywmoyue 对接口访问失败问题的修复**

## 🐛 Bug 修复

- fix: 修复扫码登录及部分常用接口663响应错误 @ywmoyue (#1632)
- 更新设置控件及引用更新 @Richasy (#1634)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2302.1.0'>情人节更新（2023）</a>.</em>

**Comment by slackingfred** [comment]
感谢修复，之前挂掉我还以为是被官方封杀了😆

**Comment by youtehub** [comment]
可以啊，好久没更新整个UI风格都改了，要是有打赏通道，可以小小支持一波
